powershell_kernel/kernel.py

```diff
@@ -7,6 +7,8 @@
 import re
 import signal
 import urllib
+import sys
+import traceback
```

Comment:
Just verifying - does this import work in all supported python versions?

Reply:
Yes, traceback is supported back to at least 2.7.